tutorials/attribution/setting-up-via-console.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Setting up via Snowplow Console
+position: 5
+---
+
+# Setting up via Snowplow Console
+
+Snowplow provides a fully managed service for running data models. We recommend this if you are just getting started out, or don’t anticipate needing to build your own custom models. To get started, follow these steps:
+
+1. Navigate to the Data models section of the Snowplow Console and click `Add data model`
+![](./screenshots/F8E9BF8C-47CF-48B7-9EDC-78D6EB1221D0_1_201_a.jpeg)
+
+2. Select `Attribution`
+![](./screenshots/Screenshot01.png)
+
+3. Set a name, warehouse connection and owner who will receive failure alerts.
```

Comment:
Is there a way to force the image to show below the text, its showing up next to it and looks weird.
![Screenshot 2024-11-28 at 16 49 43](https://github.com/user-attachments/assets/03d94757-6ce8-4c1f-a3e5-ee420b7109e1)

Reply:
It renders fine for me, maybe it is browser related..

Reply:
It seems to be anything over 2140px wide and the image appears on the same line - so is likely just the monitor I am using.

Reply:
I've added a line between the screenshot image link and the title and it seems to have fixed it.